doc/getting-started/apl-nuc.rst

```diff
@@ -84,6 +93,10 @@ complete this setup.
       | kernel-pk          | Run the Intel "PK" kernel(product kernel source)  |
       |                    | and enterprise-style kernel with backports        |
       +--------------------+---------------------------------------------------+
+      | desktop            | desktop application, Weston included              |
```

Comment:
At this point folks may not know what ``weston`` is, so a brief description would be good:

    graphical desktop application, weston
    compositing window manager included